### ISSUE TITLE
Segment multiplicity differences

### ISSUE THREAD

**Issue by bsathian** [SegmentLinking/TrackLooper#9]
We see different segment multiplicities for event 5 (and other subsequent events) when we run the code multiple times

**Comment by pwittich** [comment]
Does the number change on every run or does it follow a pattern?

**Comment by beiwang2003** [comment]
Yes, the number changes every time. No specific pattern found. Here is an example output of 3 runs (# of segments produced 57722, 57727,57713).

**Comment by slava77** [comment]
is this issue still present or was it resolved?
